src/screens/Technology/components/Details/index.tsx

```diff
@@ -279,7 +282,39 @@ export const Documents = () => {
         presentationStyle="overFullScreen"
       />
 
-      {false && <S.CostSection>Vídeos</S.CostSection>}
+      {technology.attachments.documents.length ? (
+        <>
+          <S.CostSection style={{ marginTop: 30 }}>Vídeos</S.CostSection>
+
+          {videos.map((video, idx) => (
+            <View key={idx} style={{ marginBottom: 0, backgroundColor: '#ddd' }}>
```

Comment:
It's not a good practice use the array index as an item key. It's better to use a static identifier like the video id.

```suggestion
          {videos.map((video) => (
            <View key={video.videoId} style={{ marginBottom: 0, backgroundColor: '#ddd' }}>
```

https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
NIce! Done.